.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
+
+install:
+  - cd lib/goose
+  - go get -t
+  - go build
+  - cd ../..
```

Comment:
`lib/goose` is built on L#19. I understand the using `cd` can cause issues, however the project was setup to work and build using it. I'd be down for completely reworking the structure and build process after we have something that works as it did before.